docs/variants/novacustom_nv4x_adl/releases.md

```diff
@@ -13,6 +13,81 @@ For details about our release process please read
+- [Video on USB-C docks sometimes fails to initialize](https://github.com/Dasharo/dasharo-issues/issues/560)
+- [USB-C chargers may be overloaded in certain scenarios](https://github.com/Dasharo/dasharo-issues/issues/619)
+- [Hybrid Power Boost mode of battery charger is not functional](https://github.com/Dasharo/dasharo-issues/issues/620)
+- [Laptop may shut down when disconnecting power supply](https://github.com/Dasharo/dasharo-issues/issues/621)
+- [Input current limits are set too high](https://github.com/Dasharo/dasharo-issues/issues/622)
+- [Keyboard backlight does not turn off immediately when going to sleep](https://github.com/Dasharo/dasharo-issues/issues/623)
+- [Power LED blinks erratically in HPB mode](https://github.com/Dasharo/dasharo-issues/issues/624)
+- [EC applying power limits may sometimes fail](https://github.com/Dasharo/dasharo-issues/issues/625)
+- [Wrong power limits are applied on AC attach / detach](https://github.com/Dasharo/dasharo-issues/issues/626)
+- [USB-PD controller may hang in glitched state on shutdown](https://github.com/Dasharo/dasharo-issues/issues/627)
+- [Yellow bangs in device manager](https://docs.dasharo.com/unified/clevo/post-install/#unrecognized-usb-controller-device-in-device-manager)
+- [Random freezes in Windows 11](https://github.com/Dasharo/dasharo-issues/issues/628)
+- [Noise caused by coil whine](https://github.com/Dasharo/dasharo-issues/issues/628)
+- [Low level interfering crackling/popping of the speakers while playing no sound](https://github.com/Dasharo/dasharo-issues/issues/224)
+- [ACPI error during boot (system76_acpi::kbd_backlight)](https://github.com/Dasharo/dasharo-issues/issues/559)
+- [Thunderbolt may not work after wakeup from sleep](https://github.com/Dasharo/dasharo-issues/issues/629)
+- [Unnecessary security warning after firmware update](https://github.com/Dasharo/dasharo-issues/issues/556)
+- [Early boot DMA protection setting in UEFI does not work correctly](https://github.com/Dasharo/dasharo-issues/issues/630)
+
+### Known issues
+
+- [Poor Package C-state residency at idle after first s2idle cycle](https://github.com/Dasharo/dasharo-issues/issues/631)
+- [Some commands in dasharo_ectool fail](https://github.com/Dasharo/dasharo-issues/issues/648)
+- [Incomplete serial console output](https://github.com/Dasharo/dasharo-issues/issues/614)
+- [Some devices on dock are not detected after warmboot or reboot](https://github.com/Dasharo/dasharo-issues/issues/632)
+- [Function Lock setting is not saved after cold boot](https://github.com/Dasharo/dasharo-issues/issues/458)
+- [No ability to change active PCR banks with TPM PPI in FW](https://github.com/Dasharo/dasharo-issues/issues/521)
+- [Turning wifi/bt off inside BIOS leads to an ACPI error message while system start on Gentoo Linux](https://github.com/Dasharo/dasharo-issues/issues/638)
+- [The laptop does not automatically wake up from hibernation using rtcwake](https://github.com/Dasharo/dasharo-issues/issues/485)
+
```

Comment:
We also need to add: Power state after power fail option support issue: https://github.com/Dasharo/dasharo-issues/issues/524

Reply:
Added https://github.com/Dasharo/docs/pull/729/commits/94dbdaba1e68c6cae2dd1fdd4fbdf6b85a7de5e3